pkg/elemental/pages/index.vue

```diff
@@ -101,36 +191,162 @@ export default {
+        {{ t("product.elemental") }}
+      </h1>
+      <p
+        class="description"
+        v-html="t('product.description', {}, true)"
+      />
+      <Banner
+        class="mt-40"
+        color="warning"
+        v-html="t('product.notInstalled', {}, true)"
+      />
+    </div>
     <div v-else>
+      <Banner
+        v-if="isElementalOpNotInstalledAndHasSchema"
+        class="mb-20"
+        color="warning"
+        v-html="t('product.notInstalled', {}, true)"
+      />
       <h1 class="title">
-        {{ t('elemental.menuLabels.dashboard') }}
+        {{ t('elemental.menuLabels.titleDashboard') }}
       </h1>
+      <!-- Top cards -->
       <div class="main-card-container">
         <div
           v-for="card, index in cards"
           :key="index"
           class="card"
         >
           <div class="card-top-block">
-            <h2>{{ card.count }}</h2>
+            <h1>{{ card.count }}</h1>
             <p>{{ card.title }}</p>
           </div>
-          <span>{{ card.desc }}</span>
           <button
+            v-if="card.type !== machineInvCrd || (card.type === machineInvCrd && !card.count)"
             type="button"
-            class="btn role-primary"
+            class="btn role-secondary"
             :class="{disabled: card.btnDisabled}"
             @click="handleRoute(card)"
           >
             {{ card.btnLabel }}
           </button>
+          <div
+            v-else
+            class="used-percentage-container mt-10"
+          >
+            <div>
+              <p>Used<span>{{ used }}</span></p>
+              <p>Free<span>{{ free }}</span></p>
+            </div>
+            <PercentageBar
+              class="mt-10"
+              :value="percentageBarValue"
+              :color-stops="colorStops"
+            />
+          </div>
+        </div>
+      </div>
+      <!-- Tables -->
+      <div class="main-tables-container mb-40 mt-40">
+        <div class="table-list">
+          <div class="table-title-block">
+            <h3 class="mb-20">
+              {{ machineRegTitle }}
+            </h3>
+            <nuxt-link :to="machineRegListLocation">
+              {{ t('elemental.dashboard.manageReg') }}
+            </nuxt-link>
+          </div>
+          <div
+            v-if="machineRegRows.length === 0"
+            class="empty-table-state"
+          >
+            <p>{{ t('elemental.dashboard.noMachineReg') }}</p>
+            <nuxt-link :to="machineRegCreateLocation">
+              {{ t('elemental.dashboard.noMachineRegAction') }}
+            </nuxt-link>
+          </div>
+          <ResourceTable
+            v-else
+            :rows="machineRegRows"
+            :headers="machineRegHeaders"
+            :search="false"
+            :table-actions="false"
+            :row-actions="true"
+            key-field="key"
+          >
+            <template #col:download="{col, row}">
+              <td>
+                <a class="link" @click="downloadMachineReg(row)">
```

Comment:
Feels like these should have some kind of download icon (`icon-download`) to differentiate it from the links. Not sure if that's been debated and to reduce repetition not used?

Reply:
Will need to discuss with Ken. Thanks 🙏

Reply:
Chatted with Ken, `AsyncButton` has been implemented without icon